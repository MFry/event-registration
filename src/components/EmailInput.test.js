import React from "react";
import { render } from "enzyme";
import EmailInputField from "./EmailInput";
import { ExpansionPanelActions } from "@material-ui/core";

it("Should match snapshot", () => {
  const wrapper = render(<EmailInputField />);
  expect(wrapper).toMatchSnapshot();
});
