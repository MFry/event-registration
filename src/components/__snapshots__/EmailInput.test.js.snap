// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<div
  aria-describedby="email-input-field"
  class="MuiFormControl-root-2 EmailInputField-registration-1"
>
  <div
    class="MuiFormControl-root-2"
  >
    <label
      class="MuiFormLabel-root-11 MuiInputLabel-root-6 MuiInputLabel-formControl-7 MuiInputLabel-animated-10"
      data-shrink="false"
    >
      Email
    </label>
    <div
      class="MuiInput-root-16 MuiInput-formControl-17 MuiInput-underline-20"
    >
      <input
        aria-invalid="false"
        aria-required="false"
        class="MuiInput-input-24 MuiInput-inputType-27"
        type="email"
      >
        
      </input>
    </div>
  </div>
</div>
`;
